src/controllers/admin/webhook.ts

```diff
@@ -295,15 +294,12 @@ export class WebhookController {
 		operation: OperationNameEnum,
 		logToHelper: LogToHelper,
 		userLogtoId: string,
-		productName: string
+		productId: string
 	) {
-		const roleAssignmentResponse = await logToHelper.assignCustomerPlanRoles(
-			userLogtoId,
-			productName.toLowerCase() as SupportedPlanTypes
-		);
+		const roleAssignmentResponse = await logToHelper.assignCustomerPlanRoles(userLogtoId, productId);
 		if (roleAssignmentResponse.status !== 201) {
 			const message = EventTracker.compileBasicNotification(
-				`Failed to assign roles to user for planType ${productName}: ${roleAssignmentResponse.error}`,
+				`Failed to assign roles to user for planType ${productId}: ${roleAssignmentResponse.error}`,
```

Comment:
Is this message surfaced to the user? If so, we should probably not be showing internal product plan IDs in user messages. I believe the DB contains plan IDs as well as names, you should show the friendly plan NAME based on the ID that was assigned.

Reply:
we are not persisting plan ids at the moment. But I can definitely pass the name here.